samples/system-test/detect.test.js

```diff
@@ -297,3 +298,27 @@ test(`should extract text from pdf file`, async t => {
   );
   t.true(output.includes('pdf-ocr.pdf.json'));
 });
+
+test(`should detect 'Duck', 'Bird' and 'Toy' in duck_and_truck.jpg`, async t => {
+  const output = await tools.runAsync(
+    `${cmd} localize-objects ${files[8]}`,
+    cwd
+  );
+  t.true(
+    output.includes(`Name: Bird`) &&
+      output.includes(`Name: Duck`) &&
+      output.includes(`Name: Toy`)
+  );
+});
+
+test(`should detect 'Bird'in duck_and_truck.jpg in GCS bucket`, async t => {
```

Comment:
type `'Bird'in duck`

Maybe make this consistent with the existing tests

`should detect objects in a local file`

`should detect objects in a remote file`

Reply:
Ah, just noticed these weren't named by you!

Still: would be a good change to add the consistency :)

Reply:
Fixed in latest commit.